### ISSUE TITLE
startScan not called after Android P

### ISSUE THREAD

**Issue by BenoitDuffez** [alternadom/WiFiFlutter#138]
Hi, thanks for this lib this is amazing.

I have noticed that `startScan` is not called after P:

```java
if (Build.VERSION.SDK_INT < Build.VERSION_CODES.P)
    moWiFi.startScan();
```

After reading the Android docs[1], it seems we can still call this even on P and after. I think it could help rescan the available AP at least once so that we have a refreshed list of AP.
What do you think?

1: https://developer.android.com/guide/topics/connectivity/wifi-scan They don't mention that `startScan` shouldn't be called on P and after, they just say there will be throttling.

**Comment by daadu** [comment]
This [doc](https://developer.android.com/reference/android/net/wifi/WifiManager#startScan()) suggests it is deprecated, that is why we removed it.

Can you test if it works in >= P, will add it if you confirm it.

**Comment by daadu** [comment]
Reading the doc, you mentioned further suggest it is not completely deprecated, but throttled. In that case we should remove this condition.

Can you file PR, that you have tested properly - with this removed?

**Comment by BenoitDuffez** [comment]
It's hard to test in my opinion, I never know when I'm getting the AP list from a previously cached scan or from an actively triggered fresh scan. Do you know?

**Comment by daadu** [comment]
Nope, there isnt.

By test I ment it is not crashing or throwing error, if called multiple times - instead silently ignored by Android.

**Comment by BenoitDuffez** [comment]
Okay so yes I tested on my pixel 3 xl, it doesn't crash.

**Comment by daadu** [comment]
Ok, can you file PR for this?

**Comment by daadu** [comment]
Closing this issue as fixed by #146

**Comment by daadu** [comment]
This fix/feature has landed with [v0.3.3 on pub.dev](https://pub.dev/packages/wifi_iot/versions/0.3.3).
